### ISSUE TITLE
Very bad image quality on new phone / Nothing one

### ISSUE THREAD

**Issue by mgrn0** [ethereal-developers/OpenScan#94]
I used this app on a Oneplus One Pro (7) with Android 12 and it workes flawlessly like a breeze. I recently switched to a Nothing phone one, and both with Android 12 as well as 13 the images are way smaller resolutionwise - exported images (with setting 'high') are around 1/4th of the files compared with the ones coming out of the Oneplus.

Is there a setting I did miss or should / could change? 

Problem is all resulting images or pdfs are too small, especially printed documents are hard to read.

**Comment by vikram0230** [comment]
Frankly, I do not know why exported image is of lesser quality for the same image. 
ps: Couldn't test it as we don't have Nothing One.

Did you try importing the images from gallery? Let me know if the blurry preview image issue persists.

**Comment by mgrn0** [comment]
No, using a picture from the gallery works perfectly - but I cannot set / edit the edges of the document, then.
